keanu-project/src/main/java/io/improbable/keanu/tensor/INDArrayShim.java

```diff
@@ -27,17 +35,52 @@
  */
 public class INDArrayShim {
 
+    private static final DataBuffer.Type BUFFER_TYPE = DataBuffer.Type.DOUBLE;
```

Comment:
Oh, I wanted to ask you guys about it. Would we always use `DataBuffer.Type.DOUBLE`? Or should I keep it more open and pass the buffer type in for methods where it is required?

Reply:
I can't see any methods where you use it in `INDArrayShim`?

Reply:
I'm currently passing it in as a parameter for several methods, and it's `DataBuffer.Type.DOUBLE` for all of them. So I was thinking I could just pull it out to a static variable, if we know that it will always be `DataBuffer.Type.DOUBLE`?

Reply:
Meh, decided to get rid of it.